app/app.go

```diff
@@ -106,14 +112,29 @@ func (a *App) configure() error {
 	return nil
 }
 
-func (a *App) configureJaeger() error {
-	opts := jaeger.Options{
-		Disabled:    a.config.GetBool("jaeger.disabled"),
-		Probability: a.config.GetFloat64("jaeger.samplingProbability"),
-		ServiceName: a.config.GetString("jaeger.serviceName"),
+func (a *App) configureOTEL(ctx context.Context) error {
```

Comment:
You don't need to receive a `context.Context` here

also nit: I think the capitalization should be `OTel` instead of `OTEL`

Reply:
You are right! PTAL again?